eth_bloom/crypto.py

```diff
@@ -0,0 +1,13 @@
+from Crypto.Hash import keccak
+
+
+class keccak_256:
+    def __init__(self, value):
+        self.hash = keccak.new(digest_bits=256)
+        self.hash.update(value)
```

Comment:
Initializing a new class for every hash digest feels a bit heavy-weight to me. This feels like something that can be done in a function instead.